Source/WebKit/PlatformGTK.cmake

```diff
@@ -549,50 +549,64 @@ if (ENABLE_MEDIA_STREAM)
     )
 endif ()
 
-GENERATE_API_HEADERS(WebKit2GTK_HEADER_TEMPLATES
-    ${WebKit2Gtk_DERIVED_SOURCES_DIR}/webkit
-    WebKit2GTK_INSTALLED_HEADERS
+GENERATE_API_HEADERS(WebKitGTK_HEADER_TEMPLATES
+    ${WebKitGTK_DERIVED_SOURCES_DIR}/webkit
+    WebKitGTK_INSTALLED_HEADERS
     "-DPLATFORM(GTK)=1"
     "-DPLATFORM(WPE)=0"
     "-DUSE(GTK4)=$<BOOL:${USE_GTK4}>"
 )
 
-GENERATE_API_HEADERS(WebKit2WebExtension_HEADER_TEMPLATES
-    ${WebKit2Gtk_DERIVED_SOURCES_DIR}/webkit
-    WebKit2WebExtension_INSTALLED_HEADERS
+GENERATE_API_HEADERS(WebKitWebExtension_HEADER_TEMPLATES
+    ${WebKitGTK_DERIVED_SOURCES_DIR}/webkit
+    WebKitWebExtension_INSTALLED_HEADERS
     "-DPLATFORM(GTK)=1"
     "-DPLATFORM(WPE)=0"
     "-DUSE(GTK4)=$<BOOL:${USE_GTK4}>"
 )
 
+if (USE_GTK4)
+    set(WebKitGTK_ENUM_SOURCE_TEMPLATE ${WEBKIT_DIR}/UIProcess/API/gtk/WebKitEnumTypesGtk4.cpp.in)
```

Comment:
I think splitting WebKitEnumTypesGtk4.cpp.in and WebKitEnumTypesGtk3.cpp.in should not have been needed. I should just use `#ifdefs` there since it's a source file, not an installed header.

Reply:
Fixed in upcoming revision.